src/navigation/appNavigation.js

```diff
@@ -843,26 +843,26 @@ class AppFlow extends React.Component<Props, State> {
       handleSystemDefaultThemeChange,
     } = this.props;
     const { lastAppState } = this.state;
-    BackgroundTimer.clearTimeout(lockTimer);
+    // BackgroundTimer.clearTimeout(lockTimer);
     if (isPickingImage || isBrowsingWebView) return;
     // only checking if background state for logout or websocket channel close
     if (APP_LOGOUT_STATES.includes(nextAppState)) {
       // close websocket channel instantly to receive PN while in background
       stopListeningChatWebSocket();
       // close walkthrough shade or tooltips
       endWalkthrough();
-      lockTimer = BackgroundTimer.setTimeout(() => {
-        const { navigation } = this.props;
-        const pathAndParams = navigation.router.getPathAndParamsForState(navigation.state);
-        const lastActiveScreen = pathAndParams.path.split('/').slice(-1)[0];
-        const lastActiveScreenParams = pathAndParams.params;
-        updateNavigationLastScreenState({ lastActiveScreen, lastActiveScreenParams });
-        navigation.navigate(AUTH_FLOW);
-        stopListeningNotifications();
-        stopListeningIntercomNotifications();
-        updateSignalInitiatedState(false);
-        stopListeningForBalanceChange();
-      }, SLEEP_TIMEOUT);
+      // lockTimer = BackgroundTimer.setTimeout(() => {
+      const { navigation } = this.props;
+      const pathAndParams = navigation.router.getPathAndParamsForState(navigation.state);
+      const lastActiveScreen = pathAndParams.path.split('/').slice(-1)[0];
+      const lastActiveScreenParams = pathAndParams.params;
+      updateNavigationLastScreenState({ lastActiveScreen, lastActiveScreenParams });
+      navigation.navigate(AUTH_FLOW);
```

Comment:
Just thinking that removing background timer in general might cause issues if app is continuously being switched between background and foreground. I would suggest leave the timer, but remove this part and it will only stop listening for notifications if app has been in background for over 20s while maintaining logged in state.

Reply:
agree